core/src/main/java/com/linecorp/armeria/server/DefaultServiceConfigSetters.java

```diff
@@ -40,6 +41,8 @@
     @Nullable
     private String defaultServiceName;
```

Comment:
Again, this is not needed because we can set `ServiceNaming.of(String)` to `defaultServiceNaming`.